newIDE/app/src/BehaviorsEditor/NewBehaviorDialog.js

```diff
@@ -199,6 +204,7 @@ export default function NewBehaviorDialog({
           cannotBeDismissed={false}
           flexBody
           noMargin
+          fullHeight={currentTab === 'search'}
```

Comment:
We can as well consider that this dialog is always fullHeight, even if the tab of installed behaviors :)

Reply:
I guess! Will change it